Sources/Datadog/RUM/Instrumentation/Actions/UIKit/UIKitRUMUserActionsPredicate.swift

```diff
@@ -24,29 +24,41 @@ public struct RUMAction {
     }
 }
 
+#if os(tvOS)
+public typealias UIKitRUMUserActionsPredicate = UIPressRUMUserActionsPredicate
+#else
+public typealias UIKitRUMUserActionsPredicate = UITouchRUMUserActionsPredicate
+#endif
+
 /// The predicate deciding if a given RUM Action should be recorded.
 ///
-/// When the app is running, the SDK will ask the implementation of `UIKitRUMUserActionsPredicate` if any noticed user action on the target view should
+/// When the app is running, the SDK will ask the implementation of `UITouchRUMUserActionsPredicate` if any noticed user action on the target view should
 /// be considered as the RUM Action. The predicate implementation should return RUM Action parameters if it should be recorded or `nil` otherwise.
-public protocol UIKitRUMUserActionsPredicate {
+public protocol UITouchRUMUserActionsPredicate {
```

Comment:
This will be transparent for the user: `UIKitRUMUserActionsPredicate` is still defined but as a [typealias](https://github.com/DataDog/dd-sdk-ios/pull/775/files#diff-b43a7e3806ceed834ba95f26ea02aaf2d241e6575bea97ca81d619d46b68e1f7R27-R31)